lib/ex_docker_build/docker_build.ex

```diff
@@ -2,6 +2,7 @@ defmodule ExDockerBuild.DockerBuild do
   require Logger
 
   @env ~r/^\s*(\w+)[\s=]+(.*)$/
+  @flag ~r/^\s*--(\w+)[\s=]+(.*)$$/
```

Comment:
can you get rid of all these changes and rebase from #support-for-multi-stage-builds branch in the main repo?

Reply:
you can rebase your branch with mine and point this branch to my branch that way we can chain both branches and also your branch will get my changes when rebasing, because later on you will have merging issues